faculty_cli/cli.py

```diff
@@ -460,7 +482,7 @@ def server():
 
 
 @server.command(name="list")
-@click.argument("project")
+@click.argument("provided_project", required=False, metavar="PROJECT")
```

Comment:
Can we just leave this as 'project'? I think making it optional is clear enough.
```suggestion
@click.argument("project", required=False, metavar="PROJECT")
```

Reply:
I wanted to make it clear when the user specified project was being referred to. Without the project grouping this is even less of an issue though so I've reverted back to `project`